plugins/fabrik_list/inlineedit/inlineedit.js

```diff
@@ -78,11 +79,36 @@ var FbListInlineEdit = new Class({
 			this.setFocus(this.editCell);
 		}.bind(this));
 		
-		// Click outside list clears down selection
+		// Activate edit mode in clicked element
 		window.addEvent('click', function (e) {
+			
+			// check to see if clicked element is already in edit mode
+			var activeClassName	= this.activeElement.target.className;
+			var activeClasses = activeClassName.split(" ");
+			if (activeClasses[0] == e.target.id){
+				return;
+			}
+			
```

Comment:
I overlooked this. Need to check if current element already is this.activeElement (if so no need to save anything - just ignore click and return)